app/vendor/mooltipass/app.js

```diff
@@ -92,7 +97,7 @@ mooltipass.app.onMessage = function(senderId, data, callbackFunction) {
         console.log("Cancel request for reqid " + inputObject.reqid);
         
         // Cancel request only implemented in v1.1
-        if (mooltipass.util.getFirmwareFunctionalityVersionFromVersionString(mooltipass.device.version) >= "v1.1" && mooltipass.device.currentReqid == inputObject.reqid)
+        if (mooltipass.util.getFirmwareFunctionalityVersionFromVersionString(mooltipass.device.version?mooltipass.device.version:'0') >= "v1.1" && mooltipass.device.currentReqid == inputObject.reqid)
```

Comment:
Mmmm... well, it is your stuff, I still think forging it to always have a device version doesn't seem like the best path to go. I can do it (actually, did it already). So, if you confirm, I'll commit